.travis.yml

```diff
@@ -1,10 +1,22 @@
 language: java
-dist: trusty
 jdk:
-  - oraclejdk8
-  #- oraclejdk7
-  #- openjdk7
+  - openjdk8
+
+install: true
+
+cache:
+  directories:
+    - "$HOME/.m2"
+
 before_script:
-    - echo "MAVEN_OPTS='-Xmx1024m -XX:MaxPermSize=256m'" > ~/.mavenrc
+  - echo "MAVEN_OPTS='-Xmx1024m -XX:MaxMetaspaceSize=256m'" > ~/.mavenrc
+  - rm -rf $HOME/.m2/repository/org/apache/shardingsphere/elasticjob
```

Comment:
Why rm folder in before script, can `mvn clean install` do it?

Reply:
remove useless code

Reply:
It seems like still existed.

Reply:
Finished